spec/factories/host.rb

```diff
@@ -2,7 +2,7 @@
   factory :host do
     sequence(:name)     { |n| "host_#{seq_padded_for_sorting(n)}" }
     sequence(:hostname) { |n| "host_#{seq_padded_for_sorting(n)}" }
-    vmm_vendor          "vmware"
+    vmm_vendor_display  "VMware"
```

Comment:
I think the callers for setting vmm_vendor should be changed to call it with the raw value...

For example,
before: `blah.vmm_vendor = 'Microsoft'`
after: `blah.vmm_vendor = 'microsoft'`
as opposed to `blah.vmm_vendor_display = 'Microsoft'`

Then you don't need the old `#vmm_vendor=` logic at all